osds/update-gofmt.sh

```diff
@@ -0,0 +1,33 @@
+# copyright 2019 The OpenSDS Authors.

```

Comment:
I am afraid you didn't still update the copyright.

Reply:
1. Please do not add our copyright to downloaded or copied files. 
2. Please retain the same license of original code or do only according to the license.
3. Please check the license is ok to use or not. Pay spl attn to this. Otherwise it can affect project overall.
4. If we can use as packages for external lib etc...- prefer that.